python/lib/cloudutils/serviceConfigServer.py

```diff
@@ -135,9 +135,14 @@ def checkHostName():
                 self.syscfg.svo.disableService("tomcat6")
         except:
             pass
-            
+
         self.syscfg.svo.stopService("cloudstack-management")
-        if self.syscfg.svo.enableService("cloudstack-management"):
-            return True
+
+        if self.syscfg.env.noStart == False:
+            if self.syscfg.svo.enableService("cloudstack-management"):
+                return True
+            else:
+                raise CloudRuntimeException("Failed to configure %s, please see the /var/log/cloudstack/management/setupManagement.log for detail"%self.serviceName)
```

Comment:
The original message had the log message in the wrong place (I think it was pointing at `/var/log/cloudstack-management/setupManagement.log`). I changed it to the right path on Linux systems (for Ubuntu and friends). But probably the log path can be fixed as part of this commit or in another one. But in order for the replace.properties stuff to work, there is an antrun task I believe. Not sure it would currently process files in the directory that serviceConfigServer.py is in.